### ISSUE TITLE
Verbose log messages relating to config.json

### ISSUE THREAD

**Issue by BenHall** [kubernetes/minikube#4007]
With the recent v1.0.0 release, if a `config.json` doesn't exist then the user is shown a lot of log messages that can be overwhelming. 

```
$   Downloading Kubernetes v1.14.0 images in the background ...
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
>   Creating none VM (CPUs=2, Memory=2048MB, Disk=20000MB) ...
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
2019/03/27 17:12:49 Unable to read "/root/.docker/config.json": open /root/.docker/config.json: no suchfile or directory
2019/03/27 17:12:49 No matching credentials were found, falling back on anonymous
```

**Comment by BenHall** [comment]
Likewise, if it's not a valid json file

```
2019/03/27 17:21:10 Unable to parse "/root/.docker/config.json": unexpected end of JSON input
2019/03/27 17:21:10 No matching credentials were found, falling back on anonymous
2019/03/27 17:21:10 Unable to parse "/root/.docker/config.json": unexpected end of JSON input
2019/03/27 17:21:10 No matching credentials were found, falling back on anonymous
2019/03/27 17:21:10 Unable to parse "/root/.docker/config.json": unexpected end of JSON input
2019/03/27 17:21:10 No matching credentials were found, falling back on anonymous
2019/03/27 17:21:10 Unable to parse "/root/.docker/config.json": unexpected end of JSON input
2019/03/27 17:21:10 No matching credentials were found, falling back on anonymous
```

**Comment by afbjorklund** [comment]
Was this when using the none driver as root, or when does it happen ?

**Comment by BenHall** [comment]
@afbjorklund Yep

**Comment by tstromberg** [comment]
I hit this with macOS without Docker installed as well. I'd love to see it fixed.

**Comment by afbjorklund** [comment]
It's yet another github.com/google/go-containerregistry feature. It logs all 3 of these things:

`pkg/authn/keychain.go:          log.Printf("Unable to read %q: %v", file, err)`

`pkg/authn/keychain.go:          log.Printf("Unable to parse %q: %v", file, err)`

`pkg/v1/remote/options.go:			log.Println("No matching credentials were found, falling back on anonymous")`

**Comment by afbjorklund** [comment]
Related to https://github.com/google/go-containerregistry/issues/390

**Comment by afbjorklund** [comment]
But as long as you have your docker credentials* set up properly, none of this is shown...

* https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Should be fixed (upstream) anyway of course, but that is why I didn't see it myself for instance.

``` json
{
	"experimental": "enabled",
	"credsStore": "secretservice"
}
```

**Comment by aseempatni** [comment]
I hit the same issue today on Oracle Linux using none driver as root.

**Comment by locojuhi** [comment]
I've just installed virtualbox, kubectl and minikube... but when I run `minikube start` it freeze when is downloading the minikube iso

😄  minikube v1.0.0 on linux (amd64)
🤹  Downloading Kubernetes v1.14.0 images in the background ...
2019/04/22 22:19:13 No matching credentials were found, falling back on anonymous
🔥  Creating virtualbox VM (CPUs=2, Memory=2048MB, Disk=20000MB) ...
2019/04/22 22:19:13 No matching credentials were found, falling back on anonymous
💿  Downloading Minikube ISO ...
2019/04/22 22:19:13 No matching credentials were found, falling back on anonymous
2019/04/22 22:19:13 No matching credentials were found, falling back on anonymous
2019/04/22 22:19:13 No matching credentials were found, falling back on anonymous
2019/04/22 22:19:13 No matching credentials were found, falling back on anonymous
2019/04/22 22:19:13 No matching credentials were found, falling back on anonymous
2019/04/22 22:19:13 No matching credentials were found, falling back on anonymous
2019/04/22 22:19:13 No matching credentials were found, falling back on anonymous
 2.25 MB / 142.88 MB [>------------------------------------------]   1.57% 9m36s

I haven't found any solution so far... any suggestion?

**Comment by Guojunzhou-git** [comment]
eh, It occurred to me too, and it's more difficult for me in China because "the Great Network Wall".
so do we have a convenient download way for Chineses?

**Comment by tdfacer** [comment]
I was attempting to set this up on my Arch Linux machine and found that the `minikube` package installed from the official `community` repo resulted in the "No matching credentials were found, falling back on anonymous" error.  Instead, I installed the `minikube-bin` package from the AUR and was successful in getting my local environment set up.  Before switching I ran a `rm -rf ~/.minikube` for good measure (since it wasn't automatically removed by running `sudo pacman -Rns minikube`).
